cameraClass.py

```diff
@@ -0,0 +1,36 @@
+from componentClass import Component
+from subprocess import call
+import socket
+import time
+import json
+import os
+
+class Camera(Component):
+
+    def setup(self):
+        self.sampInterval = 100.0
```

Comment:
Shouldn't this sampInterval be a parameter?

Reply:
Added the functionality to configure the sampling rate in the configuration file